slackbot-api/api/models/Message.py

```diff
@@ -3,8 +3,8 @@
 
 
 class Message(models.Model):
-    user_id = models.ForeignKey(User, on_delete=models.PROTECT)
-    channel_id = models.ForeignKey(Channel, on_delete=models.PROTECT)
+    user = models.ForeignKey(User, to_field='user_id', on_delete=models.PROTECT)
```

Comment:
i would keep it as  user_id and channel_id

Reply:
its bad practice to do so.
The ORM will replace the fields that reference foreign_keys with fields from the other Models
it seems best to just name the reference itself as the field name

Reply:
more explicitly, per instance, the ORM will need an id to be passed for referencing the table (the 'to_field' argument) when creating a new one. since we are looking to pass around only IDs for creating this behavior is fine.
When accessing the data of a given Message, it'd be misleading to say Message.user_id.foo_bar when your looking to actually access the User for said Message.

under the hood, the DB is still behaving the intended way of only having the actual 'id' (not the incremented pk) be saved for this column for both tables while still naming the column the 'to_field' argument passed which is what I consider also fine behavior.